### DISCUSSION TITLE
[spec] Mocking Date within Atom's Jasmine

### DISCUSSION THREAD

**Discussion by dannyfritz** [atom/atom#23065] [Q&A] [answered]
How do I mock the Date object within `atom test`?

I'm writing tests for the `atom-date` package and struggling to figure out how.

https://github.com/dannyfritz/atom-date/blob/github-actions/spec/commands-spec.js

**Comment by UziTech** [comment, marked as answer]
Atom uses jasmine v1.3 by default. You can use the latest version of jasmine by using [atom-jasmine3-test-runner](https://github.com/UziTech/atom-jasmine3-test-runner).

**Reply by dannyfritz** [reply]
Thanks, this looks like it will fix my issue. I'll mark this as the answer once I'm able to verify it.
